bigchaindb_driver/offchain.py

```diff
@@ -0,0 +1,325 @@
+            operations. Defaults to ``None``.
+        owners_after (:obj:`list` | :obj:`tuple` | :obj:`str`, optional):
+            One or more public keys representing the new owner(s) of the
+            asset being created or transferred. Defaults to ``None``.
+        asset (:obj:`dict`, optional): The asset being created or
+            transferred. MUST be supplied for ``'TRANSFER'`` operations.
+            Defaults to ``None``.
+        metadata (:obj:`dict`, optional): Metadata associated with the
+            transaction. Defaults to ``None``.
+        inputs (:obj:`dict` | :obj:`list` | :obj:`tuple`, optional):
+            One or more inputs holding the condition(s) that this
+            transaction intents to fulfill. Each input is expected to
+            be a :obj:`dict`. Only applies to, and MUST be supplied for,
+            ``'TRANSFER'`` operations.
+
+    Returns:
+        dict: The prepared transaction.
+
+    Raises:
+        :class:`~.exceptions.BigchaindbException`: If ``operation`` is
+            not ``'CREATE'`` or ``'TRANSFER'``.
+
+    .. important::
+
+        **CREATE operations**
+
+        * ``owners_before`` MUST be set.
+        * ``owners_after``, ``asset``, and ``metadata`` MAY be set.
+        * The argument ``inputs`` is ignored.
+        * If ``owners_after`` is not given, or evaluates to
+          ``False``, it will be set equal to ``woners_before``::
+
+            if not owners_after:
+                owners_after = owners_before
+
+        **TRANSFER operations**
+
+        * ``owners_after``, ``asset``, and ``inputs`` MUST be set.
+        * ``metadata`` MAY be set.
+        * The argument ``owners_before`` is ignored.
+
+    """
+    operation = _normalize_operation(operation)
+    return _prepare_transaction(
+        operation,
+        owners_before=owners_before,
+        owners_after=owners_after,
+        asset=asset,
+        metadata=metadata,
+        inputs=inputs,
+    )
+
+
+def prepare_create_transaction(*,
+                               owners_before,
+                               owners_after=None,
+                               asset=None,
+                               metadata=None):
+    """
+    Prepares a ``"CREATE"`` transaction payload, ready to be
+    fulfilled.
+
+    Args:
+        owners_before (:obj:`list` | :obj:`tuple` | :obj:`str`): One
+            or more public keys representing the issuer(s) of the asset
+            being created.
+        owners_after (:obj:`list` | :obj:`tuple` | :obj:`str`, optional):
+            One or more public keys representing the new owner(s) of the
+            asset being created. Defaults to ``None``.
+        asset (:obj:`dict`, optional): The asset being created. Defaults
+            to ``None``.
+        metadata (:obj:`dict`, optional): Metadata associated with the
+            transaction. Defaults to ``None``.
+
+    Returns:
+        dict: The prepared ``"CREATE"`` transaction.
```

Comment:
The reason is I used the quotes was to keep a connection with the parameter it is linked to, i.e.: `operation`.

The potential user of the library reading the docs, will pass `'CREATE'` or `'TRANSFER'`, with quotes.

That is the reasoning behind the choice to use quotes.